portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,131 @@
 <!DOCTYPE html>
 <html>
-  <head>
+    <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body>
+    </head>
+    <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+        <h1>Cynthia HU</h1>
+        <ul class="navbar">
+            <li><a href="#About">About</a></li>
+            <li><a href="#TechCorner">Tech Corner</a></li>
+            <li><a href="#TravelLog">Travel Log</a></li>
+            <li><a href="#Contact">Contact</a></li>
+        </ul>
+        <div id="About">
+            <h2>About</h2>
+            <img src="/images/profile3.jpg" alt="me in Hokkaido, Japan" 
+                class="image"
+                width="400" height="300"
+                srcset="/images/profile3.jpg 400w, 
+                        /images/profile3.jpg 800w, 
+                        /images/profile3.jpg 1200w"
+                sizes="(max-width: 400px) 200px,
+                        (max-width: 800px) 400px,
+                        (max-width: 1200px) 800px,
+                        1200px"
```

Comment:
lmk if I'm testing this wrong but it seems like the image size is the same regardless how wide my browser is

Reply:
I didn't really change this portion of the code but somehow it was working in the previous deployed website but now it is:)